YouTubeSpammerPurge.py

```diff
@@ -526,13 +526,13 @@ def confirm_continue(message=""):
   # While loop until valid input
   valid = False
   while valid == False:
-    response = input("\n" + message + " (y/n): ")
-    if response == "Y" or response == "y":
+    response = input("\n" + message + " (yes continue/no stop): ")
+    if response == "Yes keep going" or response == "Sure":
```

Comment:
How will the user know they have to type "Yes keep going"/"sure" when the prompt is "yes continue"